docs/content/iota-identity/getting-started/wasm.mdx

```diff
@@ -0,0 +1,220 @@
+---
+sidebar_label: Wasm
+description: Getting started with the IOTA Identity WASM Library.
+image: /img/identity/icon.png
+tags:
+    - identity
+    - wasm
+    - setup
+    - install
+    - getting-started
+---
+
+# Getting Started with Wasm
+
+## Minimum Requirements
+
+-   [Node.js](https://nodejs.org/en) (>= `v20`)
+
+## Install the Library
+
+If your project does not contain it already, install the peer dependency `@iota/iota-sdk` as well.
+
+```bash npm2yarn
+npm install @iota/iota-sdk
+```
+
+You can install the latest alpha version of the library by running the following command:
+
+```bash npm2yarn
+npm install @iota/identity-wasm@alpha
+```
+
+## Build the Library
+
+Alternatively, you can build the bindings yourself, e.g. if you want to want to work currently unreleased features. You can find instructions for this in `identity.rs` repository [here](https://github.com/iotaledger/identity.rs/blob/v1.6.0-alpha/bindings/wasm/identity_wasm#build).
+
+## NodeJS Usage
+
+The following code creates a new IOTA DID Document suitable for publishing to a locally running test network.  See the [instructions](./local-network-setup) on running your own test network.
```

Comment:
```suggestion
The following code creates a new IOTA DID Document suitable for publishing to a [locally running test network](./local-network-setup).
```

Reply:
Applied the other suggestion, with the same content. ^^